taskt/UI/Forms/frmSettings.cs

```diff
@@ -417,16 +387,16 @@ private void btnLaunchDisplayManager_Click(object sender, EventArgs e)
         {
             Supplemental.frmDisplayManager displayManager = new Supplemental.frmDisplayManager();
```

Comment:
Please add a using reference to `Supplemental` and cleanup this namespace from code.

Reply:
done